examples/src/main/scala/org/apache/spark/examples/streaming/ActorWordCount.scala

```diff
@@ -18,16 +18,14 @@
 // scalastyle:off println
 package org.apache.spark.examples.streaming
 
-import scala.collection.mutable.LinkedList
-import scala.reflect.ClassTag
-import scala.util.Random
-
 import akka.actor._
 import com.typesafe.config.ConfigFactory
-
 import org.apache.spark.SparkConf
-import org.apache.spark.streaming.{Seconds, StreamingContext}
 import org.apache.spark.streaming.akka.{ActorReceiver, AkkaUtils}
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+
+import scala.collection.mutable._
```

Comment:
Is there any configuration file which I can import so that intellij uses recommended settings as default ?